### ISSUE TITLE
Application Signing Error

### ISSUE THREAD

**Issue by ghost** [fgl27/smarttv-twitch#83]
Im having trouble running the file after completing all the certificate manager etc. However everytime i have tried deleting and adding the certificate (Samsung certificate, then add another author to Tizen), it still sends me the same message over and over. I may have missed one or two steps but cannot diagnose this on my own. I dont see anybody has the same problem as me, so this is quite embarassing to post an issue. Thanks 
![asdasdasdasd](https://user-images.githubusercontent.com/58338839/69880480-963f2280-12fc-11ea-9c65-951c7c4988d9.png)

**Comment by fgl27** [comment]
Tizen is very odd, some have a issue others don't.

Try the below process and let me know the result.

https://github.com/fgl27/smarttv-twitch/tree/master/release/githubio/etc

**Comment by kappachinoo** [comment]
> Tizen is very odd, some have a issue others don't.
> 
> Try the below process and let me know the result.
> 
> https://github.com/fgl27/smarttv-twitch/tree/master/release/githubio/etc

i have done redoing the samsung certificate, it still shows me the same exact message. The device manager has shown that the TV is connected and succesfully upload a certificate. But still no improvement. Should i re-install tizen studio? If so how do i do it? On my programs and features, tizen studio does not appear.

**Comment by fgl27** [comment]
Not on windows, must have a uninstaller somewhere in the app installation folder.

But is not a question to install I guess you are missing some packages...
So re start from here
https://github.com/fgl27/smarttv-twitch#tizen-package-managerinstalling-the-tools

make sure all is installed as in the images and try again.

**Comment by kappachinoo** [comment]
![123](https://user-images.githubusercontent.com/58338839/69883447-cdb2cc80-1306-11ea-99aa-8ea7d2085f1c.png)
![1234](https://user-images.githubusercontent.com/58338839/69883448-ce4b6300-1306-11ea-9b19-db31579a1150.png)
Double checked everything, the only odd thing is that on the Extension SDK, Samsuing certificate extension only shows up on the IOT-Headless 5.5 Tab, not anywhere else. For the Main SDK, i basically install everything, (i thought it would be better to install all of it than picking each one).

**Comment by fgl27** [comment]
You don't need the IOT-headless as the certificate extension is outside of it...

I have only those package installed and it works so I have no idea why that error shows...

Try to un-install re-install processed, if you don't find a uninstaller delete all tizen folders.

**Comment by kappachinoo** [comment]
Hello there, Good news!, I uninstalled all of the tizen studio app etc, and followed step by step of the guide. And surprisingly it worked right after i hit run. This time, i only made a Samsung ceritifcation account, (not the tizen one) and it worked. The only thing i might have done wrong is that on the first time, I decided to make the Tizen certification first then connecting the TV through the device manager. I think that what triggers the bug/error messages. 
Nevertheless I will watch over this forum and help others who might need in help.
Thanks you so much for your time!
